odl/discr/discr_mappings.py

```diff
@@ -380,106 +389,41 @@ def _call(self, inp):
         >>> val_arr = np.array([c for c in 'mystring'])
         >>> values = dspace.element(val_arr)
         >>> function = interp_op(values)
-        >>> val = function(0.3, 0.6)  # closest to index (1, 1) -> 3
+        >>> val = function([0.3, 0.6])  # closest to index (1, 1) -> 3
         >>> print(val)
         t
+        >>> out = np.empty(2, dtype='U1')
+        >>> pts = np.array([[0.3, 0.6],
+        ...                 [1.0, 1.0]])
+        >>> out = function(pts.T, out=out)  # returns original out
```

Comment:
Somehow lost the comment on that. It's necessary since here, I initialize point by point, which is somehow natural, but in the evaluation, I need the first axis to determine the vector components, i.e. `x[0]` must be the array of first components, not the first point. Therefore `.T`.